controllers/challenge_controller.go

```diff
@@ -0,0 +1,100 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/helpers"
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type ChallengeController interface {
+	Create(context *gin.Context)
+}
+
+type challengeController struct{}
+
+func NewChallengeController() ChallengeController {
+	return &challengeController{}
+}
+
+// TODO: Idea for future, if we want to let companies/orgs create challenges (Business Model #3).
+// Default method of adding challenges on frontend is to add by placing a marker on a map
+// (allowing us to grab Lat/Long coordinates). We can also offer a (premium) option for users to
+// provide just the address, and convert to Lat/Long -- thus allowing batch creation of challenges.
+func (ctrl *challengeController) Create(context *gin.Context) {
+	type ICreate struct {
+		Latitude    float64 `json:"latitude"`
+		Longitude   float64 `json:"longitude"`
+		Address     string  `json:"address"`
+		TaskName    string  `json:"taskName" binding:"required"`
```

Comment:
nit: I know this isn't super apart of your ticket, but i don't super like the `TaskName` naming (particularly since task is implied imo). Can we change this to just `Name` for this and also the model schema as well?

Reply:
fixed